modules/tinymce/src/plugins/lists/test/ts/browser/ContentEditableFalseActionsTest.ts

```diff
@@ -0,0 +1,96 @@
+    const editor = hook.editor();
+    const content = '<ul contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ul>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Numbered list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="Bullet list"]');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+  });
+
+  it('TINY-8920: List commands are disabled in noneditable OL list', () => {
+    const editor = hook.editor();
+    const content = '<ul contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ul>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    editor.execCommand('RemoveList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertUnorderedList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertOrderedList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertDefinitionList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('mceListProps');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+  });
+
+  it('TINY-8920: List commands are disabled in noneditable UL list', () => {
+    const editor = hook.editor();
+    const content = '<ul contenteditable="false">\n' +
+      '<li contenteditable="true">editable</li>\n' +
+      '<li>noneditable</li>\n' +
+      '<li contenteditable="true">editable</li>\n' +
+    '</ul>';
+    editor.setContent(content);
+    TinySelections.setCursor(editor, [ 1, 2, 0 ], 0);
+    editor.execCommand('RemoveList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertUnorderedList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertOrderedList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
+    editor.execCommand('InsertDefinitionList');
+    TinyAssertions.assertCursor(editor, [ 0, 2, 0 ], 0);
+    TinyAssertions.assertContent(editor, content);
```

Comment:
is `mceListProps` command applicable to UL list?

Reply:
No, that command opens a dialog which allows the choice of a starting number for an OL list.